source/js/components/petition/petition.jsx

```diff
@@ -575,12 +575,14 @@ export default class Petition extends React.Component {
                 {this.state.userTriedSubmitting && !this.refs.privacy.checked && <small className="has-danger">Please check this box if you want to proceed</small>}
               </label>
             </div>
-            <div className="my-2">
-              <label className="form-check-label">
-                <input disabled={disableFields} type="checkbox" className="form-check-input" id="PrivacyCheckbox" ref="newsletterSignup" />
-                <span className="h6-heading form-text">Yes, I want to receive email updates about Mozilla’s campaigns.</span>
-              </label>
-            </div>
+            { this.props.subscribed ? null :
+              <div className="my-2">
+                <label className="form-check-label">
+                  <input disabled={disableFields} type="checkbox" className="form-check-input" id="PrivacyCheckbox" ref="newsletterSignup" />
```

Comment:
let's just change this ID at the same time. It should be NewsletterSignup, not PrivacyCheckbox